code/collections.rb

```diff
@@ -1,15 +1,32 @@
 # collections
 
 # 1. Create a variable `point1` and assign it a hash representing a coordinate with keys `:x` and `:y` which have integer values
-
+point1 = {
+  x:1,
+  y:2,
+}
 # 2. Print the y value
-
+point[:y]
```

Comment:
this just returns, want to print `puts point[:y]`. Remember that printing return values of expressions is a feature of the RE**P**L but when we need to print from running code, we need to explicitly call `puts`